client/lib/plans/features-list.tsx

```diff
@@ -541,10 +541,10 @@ export const FEATURES_LIST: FeatureList = {
 
 	[ FEATURE_PERSONAL_THEMES ]: {
 		getSlug: () => FEATURE_PERSONAL_THEMES,
-		getTitle: () => i18n.translate( 'Unlimited personal themes' ),
+		getTitle: () => i18n.translate( 'Unlimited starter themes' ),
 		getDescription: () =>
 			i18n.translate(
-				'Unlimited access to all of our personal themes, including designs specifically tailored for businesses.'
+				'Unlimited access to all of our starter themes, including designs specifically tailored for businesses.'
```

Comment:
I don't want to block this PR on it, but the business mention surprised me. I don't think the Starter plan is generally the best fit there. Something for another day.

Reply:
The difference is case it would say "Get access to our Starter themes", which might be the right thing to do, but isn't what we were doing before.

I think we should probably put this PR on hold until paYJgx-4vW-p2 is resolved.